python/lsst/meas/base/sdssCentroid.cc

```diff
@@ -0,0 +1,68 @@
+/* 
+ * LSST Data Management System
+ * Copyright 2008-2016  AURA/LSST.
+ * 
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the LSST License Statement and 
+ * the GNU General Public License along with this program.  If not, 
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include <pybind11/pybind11.h>
+//#include <pybind11/stl.h>
+
+#include "lsst/pex/config/pybind11.h"
+#include "lsst/meas/base/pybind11.h"
+
+#include "lsst/meas/base/SdssCentroid.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace base {
+
+PYBIND11_PLUGIN(_sdssCentroid) {
+    py::module mod("_sdssCentroid", "Python wrapper for afw _sdssCentroid library");
+
+    /* Module level */
+    py::class_<SdssCentroidAlgorithm, SimpleAlgorithm> clsSdssCentroidAlgorithm(mod, "SdssCentroidAlgorithm");
+    py::class_<SdssCentroidControl> clsSdssCentroidControl(mod, "SdssCentroidControl");
+    py::class_<SdssCentroidTransform> clsSdssCentroidTransform(mod, "SdssCentroidTransform");
+
+    /* Member types and enums */
+    clsSdssCentroidAlgorithm.def(py::init<SdssCentroidAlgorithm::Control const &,
+                                          std::string const &,
+                                          afw::table::Schema &>(),
+                                 "ctrl"_a, "name"_a, "schema"_a);
+
+    /* Constructors */
+
+    /* Operators */
+
+    /* Members */
+    declareAlgCtrlXfrm<SdssCentroidAlgorithm,
```

Comment:
I'm not sure I like abbreviations, but I guess in this case the full name is really too long.